### ISSUE TITLE
Some questions about reproduce details

### ISSUE THREAD

**Issue by HaiyangLiu1997** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#109]
Firstly, Thanks for your excellent work! 

I have some questions about the reproduce of openpose model.
you know the openpose has it new network, I have written one by myself, and I found that in your network folders there also one openpose new model.
have you ever try to train it and what about the accuracy?
when I try it I found two problems:
1. it often occur loss becoming nan or inf when start training. reduce lr can work, instead of reducing lr, I add one 1*1 conv layer in each densebolck, (ex: transfer the channels from 384 to 128). 
    I don't find the new openpose source model, is your openpose model absolutely as same as openpose?
    for the rtpose model you attached, do you use the default parameters in your code? such as lr = 1, decay = 0. and only use 56000 images?  usually, we division the lr with some factors in the 2/3 period,  I found you use auto lr control function to realize it. do you remember how many its or epochs it needed or can you attach the tensorboad log file? I am sorry to bother you, if it is not convenient to share, it doesn't matter because you already help us a lot.
2. the rtpose model usually will get average heatmap loss 0.0010 and paf loss 0.0015 when converge, so I guess for the new openpose model, when the heatmap and paf get the similar average loss(ex: 0.0085,i use 5 paf and 1 heatmap, 0.0075 + 0.0010), it should get a similar accuracy.
    however, the new model will get only 0.33mAP,for single scale, which should be at least 0.48 for single scale old openpose model.
    have you ever try to train the new openpose model, could I ask have you meet these problems? 
     Thank you very much and looking forward for your reply.
